pep-0458.txt

```diff
@@ -828,6 +828,14 @@ version of the *snapshot* metadata, which in turn lists the versions of the
 *targets* and delegated targets metadata, all as part of a given consistent
 snapshot.
 
+In normal usage, version number overflow is unlikely to occur as an 8-byte
+integer can be incremented once per millisecond and last almost 300 million
```

Comment:
Minor nit: I'd not take it for granted that  an 8-byte integer is chosen. Maybe we can phrase this more like a recommendation. Something like:

*"... unlikely to occur. An 8-byte unsigned integer, for instance, can be
incremented once per millisecond for almost 300 million years."*

What do you think?

Reply:
I agree, this was intended mostly as an example. I'll make that more clear.